### PR TITLE
Add modular code for profiling of sql queries on related pages and structured some code

### PR DESCRIPTION
**Pull request by Piyush3079**
This pr contains the functional code for sql profiling. Also this or fixes some of the bugs of modular code.
This pr is in continuation of Mod_Js_Sql_Js. Before reviewing this pr, review and merge Mod_Js_Sql_js.

Before submitting pull request, please check that every commit:

- [ ] Has proper Signed-Off-By
- [ ] Has commit message which describes it
- [ ] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [ ] Any new functionality is covered by tests

### CODE REVIEW

**Comment by devenbansod** [comment]
@Piyush3079 Please rebase

File: js/src/index.js [hunk 1]
```diff
@@ -111,7 +113,7 @@ $(document).ajaxError(function (event, request, settings) {
  * Adding common files for every page
  */
 for (let i in files.global) {
-    AJAX.scriptHandler.add(files.global[i]);
+    AJAX.scriptHandler.add(files.global[i], 1);
```
Comment (devenbansod) [inline js/src/index.js]: Why this change?

**Review by devenbansod — APPROVED** [review]
Minor comment. Otherwise LGTM

File: js/src/index.js (hunk 1 above)
Comment (Piyush3079) [reply js/src/index.js]: This actually taking care of adding scripts in the `_scriptsToBeLoaded`. If we don't send 1, the scripts will not be added in the object and on making ajax request the same script will be loaded again without calling teardown for the previous script, causing multiple events for that page.
